OWML.ModHelper/Logging/ConsoleUtils.cs

```diff
@@ -0,0 +1,65 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace OWML.ModHelper
+{
+    public static class ConsoleUtils
+    {
+        public static readonly List<String> ErrorList = new List<String> {
+            "error",
+            "exception"
+        };
+
+        public static readonly List<String> WarningList = new List<String> {
+            "warning",
+            "disabled"
+        };
+
+        public static readonly List<String> SuccessList = new List<String> {
+            "success"
+        };
+
+        public static void WriteByType(MessageType type, string line)
+        {
+            switch (type)
+            {
+                case MessageType.Error:
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    break;
+                case MessageType.Warning:
+                    Console.ForegroundColor = ConsoleColor.Yellow;
+                    break;
+                case MessageType.Success:
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    break;
+                case MessageType.Message:
+                    Console.ForegroundColor = ConsoleColor.Gray;
+                    break;
+                case MessageType.Info:
+                    Console.ForegroundColor = ConsoleColor.Cyan;
+                    break;
+            }
+            Console.WriteLine(line);
+            Console.ForegroundColor = ConsoleColor.Gray;
+        }
+
+        public static MessageType ContentsToType(string line)
```

Comment:
if this method is only used one place there's no need to move it from that place?

Reply:
Actually, this is another reason i hate static classes. They're accessable from anywhere, so they tend to be used from anywhere, and gradually become trash cans for random code that don't really belong together. It's a slippery slope, you won't believe the things I've seen 😋
So - if this method is only used from one place - move it (back) to that place ;-)